app/controllers/posts_controller.rb

```diff
@@ -2,11 +2,17 @@ class PostsController < ApplicationController
   # Create action - Handle the creation of a new post
 
   def index
-    @post = Post.all
+    @post = Post.where(author_id: params[:user_id])
+    @comment_recent = Comment.all.order(created_at: :desc).limit(5)
```

Comment:
- [ ] Great job so far @francksefu However, as the previous reviewers  mentioned instead of creating a new method to return the `recent comments` in the Posts `index.html.erb` file, kindly remove this method and make use of this method that you created initially in the previous milestone in the `Post Model` instead :+1: 
- [ ] I have pasted the method below :+1: 

[WebApp/app/models/user.rb](https://github.com/francksefu/WebApp/blob/a63e0e7a861dd92dac0a0fb3f3d3832d675f3d65/app/models/user.rb#L6-L9)
```ruby
 def recent_post 
   Post.where(author_id: id).limit(3).order(created_at: :desc) 
 end
```